mlrun/model_monitoring/db/tsdb/base.py

```diff
@@ -103,8 +106,8 @@ def get_model_endpoint_real_time_metrics(
     def get_records(
         self,
         table: str,
-        start: str,
-        end: str,
+        start: typing.Union[datetime, str],
+        end: typing.Union[datetime, str],
```

Comment:
```suggestion
        start: datetime,
        end: datetime,
```
Since you support only `datetime` and not `str` in TDEngine. It's ok to extend the supported types in subclasses, but not to narrow them.